lib/govuk_publishing_components/presenters/select.rb

```diff
@@ -0,0 +1,42 @@
+module GovukPublishingComponents
+  module Presenters
+    class SelectHelper
+      attr_reader :options, :option_markup, :selected_option
+
+      def initialize(local_assigns)
+        @options = local_assigns[:options] if local_assigns[:options]
```

Comment:
`if local_assigns[:options]` isn't required here. It's better that `options` is nil rather than being undefined.